src/components/buttons/PrimaryOutlineBtn.tsx

```diff
@@ -0,0 +1,23 @@
+import { buttonProps } from "./button.type";
+
+export const PrimaryOutlineBtn = ({
+  children,
+  type = "button",
+  cnames,
+  onClick,
+  disable,
+}: buttonProps) => {
+  return (
+    <button
+      className={
+        "py-2 px-2.5 rounded text-black border border-primary hover:bg-primary hover:text-white transition-colors	 " +
```

Comment:
What does '+' imply in classname? Got to know something new

Reply:
I am just concatenating the strings there.